consensus/clique/src/main/java/org/hyperledger/besu/consensus/clique/CliqueProtocolSchedule.java

```diff
@@ -72,18 +78,36 @@ public static ProtocolSchedule create(
 
     final EpochManager epochManager = new EpochManager(cliqueConfig.getEpochLength());
 
+    final Map<Long, Function<ProtocolSpecBuilder, ProtocolSpecBuilder>> specMap = new HashMap<>();
+    forksSchedule
+        .getForks()
+        .forEach(
+            forkSpec ->
+                specMap.put(
+                    forkSpec.getBlock(),
+                    builder ->
+                        applyCliqueSpecificModifications(
+                            epochManager,
+                            forkSpec.getValue().getBlockPeriodSeconds(),
+                            cliqueConfig.getCreateEmptyBlocks(),
+                            localNodeAddress,
+                            builder)));
+    final ProtocolSpecAdapters specAdapters = new ProtocolSpecAdapters(specMap);
+
+    ProtocolSpecAdapters.create(
+        0,
+        builder ->
+            applyCliqueSpecificModifications(
+                epochManager,
+                cliqueConfig.getBlockPeriodSeconds(),
+                cliqueConfig.getCreateEmptyBlocks(),
+                localNodeAddress,
+                builder));
```

Comment:
@jframe Think this is redundant and could/should be removed?

Reply:
Looks redundant to me